frontend/packages/console-shared/src/components/dashboard/activity-card/activity-card.scss

```
@@ -55,6 +55,11 @@
   }
 }
 
+.co-activity-card__recent-actions.pf-c-button.pf-m-link.pf-m-inline {
```

Does this style need to be so specific? I'd suggest

```suggestion
.co-activity-card__recent-actions {
```

or add another class with a BEM modifier if we need to restrict it to certain places.

---

We dont need to restrict it, the style is used only in one place but PF4 is using styled components so If I just pass `co-activity-card__recent-actions` to PF4 button, my rules will still get overridden by theirs. Therefore I need to be more specific to get the priority in styling.

---

I'd just use `!important` in that case. We're overriding something and we always want this rule to take precedence, so it makes sense here.

---

Ok, if `!important` is fine I can actually go with a bit different approach and use `icon` prop of PF4 button. Currently there's a bug on PF side which causes the icon to be misaligned with I can fix that with `!important`

Can you take a look again ?